### PR TITLE
Adding Health Monitor Resource for UDP and POSTGRESQL

### PR DESCRIPTION
**Pull request by GaddamSaketha**

### CODE REVIEW

File: bigip/resource_bigip_ltm_monitor_test.go [hunk 1]
```diff
@@ -144,6 +168,46 @@ func TestAccBigipLtmMonitor_create(t *testing.T) {
 			},
 		},
 	})
+	resource.Test(t, resource.TestCase{
```
Comment (RavinderReddyF5) [inline bigip/resource_bigip_ltm_monitor_test.go:171]: Can you please add Acceptance Tests results for confirmation

File: vendor/github.com/f5devcentral/go-bigip/ltm.go [hunk 2]
```diff
@@ -2388,7 +2388,7 @@ func (b *BigIP) DeleteVirtualAddress(vaddr string) error {
 // Monitors returns a list of all HTTP, HTTPS, Gateway ICMP, ICMP, and TCP monitors.
 func (b *BigIP) Monitors() ([]Monitor, error) {
 	var monitors []Monitor
-	monitorUris := []string{"http", "https", "icmp", "gateway-icmp", "tcp", "tcp-half-open", "ftp"}
+	monitorUris := []string{"http", "https", "icmp", "gateway-icmp", "tcp", "tcp-half-open", "ftp", "udp", "postgresql"}
```
Comment (RavinderReddyF5) [inline vendor/github.com/f5devcentral/go-bigip/ltm.go:2391]: Can we relook on all the supported monitors and add all those monitor in this array??

File: bigip/resource_bigip_ltm_monitor_test.go (hunk 1 above)
Comment (GaddamSaketha) [reply bigip/resource_bigip_ltm_monitor_test.go:171]: === RUN   TestAccBigipLtmMonitor_create
--- PASS: TestAccBigipLtmMonitor_create (14.52s)
=== RUN   TestAccBigipLtmMonitor_import
--- PASS: TestAccBigipLtmMonitor_import (3.89s)

Comment (GaddamSaketha) [reply bigip/resource_bigip_ltm_monitor_test.go:171]: ==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./... -v  -timeout 120m
?       github.com/terraform-providers/terraform-provider-bigip [no test files]
=== RUN   TestAccProvider
--- PASS: TestAccProvider (0.01s)
=== RUN   TestAccBigipAs3_create
--- PASS: TestAccBigipAs3_create (26.78s)
=== RUN   TestAccBigipCmDevice_create
--- PASS: TestAccBigipCmDevice_create (10.87s)
=== RUN   TestAccBigipCmDevice_import
--- PASS: TestAccBigipCmDevice_import (8.86s)
=== RUN   TestAccBigipCmDevicegroup_create
--- PASS: TestAccBigipCmDevicegroup_create (11.07s)
=== RUN   TestAccBigipCmDevicegroup_import
--- PASS: TestAccBigipCmDevicegroup_import (10.76s)
=== RUN   TestAccBigipLtmDataGroup_create
--- PASS: TestAccBigipLtmDataGroup_create (27.20s)
=== RUN   TestAccBigipLtmDataGroup_import
--- PASS: TestAccBigipLtmDataGroup_import (27.97s)
=== RUN   TestAccBigipLtmIRule_create
--- PASS: TestAccBigipLtmIRule_create (9.09s)
=== RUN   TestAccBigipLtmIRule_import
--- PASS: TestAccBigipLtmIRule_import (9.10s)
=== RUN   TestAccBigipLtmMonitor_create
--- PASS: TestAccBigipLtmMonitor_create (122.66s)
=== RUN   TestAccBigipLtmMonitor_import
--- PASS: TestAccBigipLtmMonitor_import (24.57s)
=== RUN   TestAccBigipLtmNode_create
--- PASS: TestAccBigipLtmNode_create (18.47s)
=== RUN   TestAccBigipLtmNode_import
--- PASS: TestAccBigipLtmNode_import (18.15s)
=== RUN   TestAccBigipLtmNodeInvalid
--- PASS: TestAccBigipLtmNodeInvalid (0.03s)
=== RUN   TestAccBigipLtmNodeCreate
--- PASS: TestAccBigipLtmNodeCreate (0.09s)
=== RUN   TestAccBigipLtmPersistenceProfileCookieCreate
--- PASS: TestAccBigipLtmPersistenceProfileCookieCreate (10.02s)
=== RUN   TestAccBigipLtmPersistenceProfileCookieImport
--- PASS: TestAccBigipLtmPersistenceProfileCookieImport (9.72s)
=== RUN   TestAccBigipLtmPersistenceProfileDstAddrCreate
--- PASS: TestAccBigipLtmPersistenceProfileDstAddrCreate (9.69s)
=== RUN   TestAccBigipLtmPersistenceProfileDstAddrImport
--- PASS: TestAccBigipLtmPersistenceProfileDstAddrImport (9.59s)
=== RUN   TestAccBigipLtmPersistenceProfileSrcAddrCreate
--- PASS: TestAccBigipLtmPersistenceProfileSrcAddrCreate (9.85s)
=== RUN   TestAccBigipLtmPersistenceProfileSrcAddrImport
--- PASS: TestAccBigipLtmPersistenceProfileSrcAddrImport (9.86s)
=== RUN   TestAccBigipLtmPersistenceProfileSSLCreate
--- PASS: TestAccBigipLtmPersistenceProfileSSLCreate (10.45s)
=== RUN   TestAccBigipLtmPersistenceProfileSSLImport
--- PASS: TestAccBigipLtmPersistenceProfileSSLImport (10.09s)
=== RUN   TestAccBigipLtmPolicy_create
--- PASS: TestAccBigipLtmPolicy_create (20.20s)
=== RUN   TestAccBigipLtmPolicy_import
--- PASS: TestAccBigipLtmPolicy_import (19.21s)
=== RUN   TestAccBigipLtmPool_create
--- PASS: TestAccBigipLtmPool_create (12.45s)
=== RUN   TestAccBigipLtmPool_import
--- PASS: TestAccBigipLtmPool_import (12.82s)
=== RUN   TestAccBigipLtmfasthttp_create
--- PASS: TestAccBigipLtmfasthttp_create (8.65s)
=== RUN   TestAccBigipLtmProfilefasthttp_import
--- PASS: TestAccBigipLtmProfilefasthttp_import (8.68s)
=== RUN   TestAccBigipLtmProfileFastl4_create
--- PASS: TestAccBigipLtmProfileFastl4_create (8.88s)
=== RUN   TestAccBigipLtmProfileFastl4_import
--- PASS: TestAccBigipLtmProfileFastl4_import (8.74s)
=== RUN   TestAccBigipLtmProfileHttp2_create
--- PASS: TestAccBigipLtmProfileHttp2_create (8.78s)
=== RUN   TestAccBigipLtmProfileHttp2_modify
--- PASS: TestAccBigipLtmProfileHttp2_modify (15.91s)
=== RUN   TestAccBigipLtmProfileHttp2_import
--- PASS: TestAccBigipLtmProfileHttp2_import (8.84s)
=== RUN   TestAccBigipLtmProfilehttp_create
--- PASS: TestAccBigipLtmProfilehttp_create (9.61s)
=== RUN   TestAccBigipLtmProfilehttp_import
--- PASS: TestAccBigipLtmProfilehttp_import (9.85s)
=== RUN   TestAccBigipLtmProfileHttpcompress_create
--- PASS: TestAccBigipLtmProfileHttpcompress_create (26.94s)
=== RUN   TestAccBigipLtmProfileHttpcompress_import
--- PASS: TestAccBigipLtmProfileHttpcompress_import (8.68s)
=== RUN   TestAccBigipLtmProfileoneconnect_create
--- PASS: TestAccBigipLtmProfileoneconnect_create (8.61s)
=== RUN   TestAccBigipLtmProfileoneconnect_import
--- PASS: TestAccBigipLtmProfileoneconnect_import (8.70s)
=== RUN   TestAccBigipLtmProfileTcp_create
--- PASS: TestAccBigipLtmProfileTcp_create (8.74s)
=== RUN   TestAccBigipLtmProfileTcp_import
--- PASS: TestAccBigipLtmProfileTcp_import (8.91s)
=== RUN   TestAccBigipLtmsnat_create
--- PASS: TestAccBigipLtmsnat_create (8.76s)
=== RUN   TestAccBigipLtmsnat_import
--- PASS: TestAccBigipLtmsnat_import (8.61s)
=== RUN   TestAccBigipLtmsnatpool_create
--- PASS: TestAccBigipLtmsnatpool_create (8.69s)
=== RUN   TestAccBigipLtmsnatpool_import
--- PASS: TestAccBigipLtmsnatpool_import (8.69s)
=== RUN   TestAccBigipLtmVA_create
--- PASS: TestAccBigipLtmVA_create (9.38s)
=== RUN   TestAccBigipLtmVA_import
--- PASS: TestAccBigipLtmVA_import (9.30s)
=== RUN   TestAccBigipLtmVS_create
--- PASS: TestAccBigipLtmVS_create (21.39s)
=== RUN   TestAccBigipLtmVS_create_Defaultstate
--- PASS: TestAccBigipLtmVS_create_Defaultstate (13.53s)
=== RUN   TestAccBigipLtmVS_Modify_stateDisabledtoEnabled
--- PASS: TestAccBigipLtmVS_Modify_stateDisabledtoEnabled (24.58s)
=== RUN   TestAccBigipLtmVS_Modify_stateEnabledtoDisabled
--- PASS: TestAccBigipLtmVS_Modify_stateEnabledtoDisabled (24.73s)
=== RUN   TestAccBigipLtmVS_import
--- PASS: TestAccBigipLtmVS_import (21.05s)
=== RUN   TestAccBigipNetroute_create
--- PASS: TestAccBigipNetroute_create (12.57s)
=== RUN   TestAccBigipNetroute_import
--- PASS: TestAccBigipNetroute_import (12.86s)
=== RUN   TestAccBigipNetselfip_create
--- PASS: TestAccBigipNetselfip_create (13.37s)
=== RUN   TestAccBigipNetselfip_import
--- PASS: TestAccBigipNetselfip_import (25.79s)
=== RUN   TestAccBigipNetvlan_create
--- PASS: TestAccBigipNetvlan_create (11.57s)
=== RUN   TestAccBigipNetvlan_import
--- PASS: TestAccBigipNetvlan_import (11.52s)
=== RUN   TestAccBigipSysdns_create
--- PASS: TestAccBigipSysdns_create (70.18s)
=== RUN   TestAccBigipSysdns_import
--- PASS: TestAccBigipSysdns_import (8.78s)
=== RUN   TestAccBigipSysIapp_create
--- PASS: TestAccBigipSysIapp_create (9.63s)
=== RUN   TestAccBigipSysIapp_import
--- PASS: TestAccBigipSysIapp_import (8.91s)
=== RUN   TestAccBigipSysNtp_create
--- PASS: TestAccBigipSysNtp_create (8.29s)
=== RUN   TestAccBigipSysNtp_import
--- PASS: TestAccBigipSysNtp_import (8.21s)
=== RUN   TestAccBigipSysNtpInvalid
--- PASS: TestAccBigipSysNtpInvalid (0.02s)
=== RUN   TestAccBigipSysProvision_create
--- PASS: TestAccBigipSysProvision_create (8.19s)
=== RUN   TestAccBigipSysProvision_import
--- PASS: TestAccBigipSysProvision_import (7.99s)
=== RUN   TestAccBigipSyssnmp_create
--- PASS: TestAccBigipSyssnmp_create (8.62s)
=== RUN   TestAccBigipSyssnmp_import
--- PASS: TestAccBigipSyssnmp_import (8.39s)
=== RUN   TestValidStringValue
--- PASS: TestValidStringValue (0.00s)
=== RUN   TestInvalidStringValue
--- PASS: TestInvalidStringValue (0.00s)
=== RUN   TestF5NameString
--- PASS: TestF5NameString (0.00s)
=== RUN   TestF5NameSet
--- PASS: TestF5NameSet (0.00s)
=== RUN   TestF5NameList
--- PASS: TestF5NameList (0.00s)
=== RUN   TestValidateEnabledDisabledString
--- PASS: TestValidateEnabledDisabledString (0.00s)
=== RUN   TestValidateReqPrefDisabledString
--- PASS: TestValidateReqPrefDisabledString (0.00s)
PASS
ok      github.com/terraform-providers/terraform-provider-bigip/bigip   1021.802s

**Review by RavinderReddyF5 — APPROVED** [review]

**Pull request merged**